objectivec/include/ort_session.h

```diff
@@ -183,6 +183,25 @@ NS_ASSUME_NONNULL_BEGIN
                         value:(NSString*)value
                         error:(NSError**)error;
 
+/**
+ * Registers custom ops for use with {@link ORTSession}s using this SessionOptions by calling the specified
```

Comment:
not sure if `@link` is supported by the doc renderer
https://github.com/realm/jazzy#supported-documentation-keywords

seems like `` `OrtSession` `` should work